### ISSUE TITLE
Make PluginPanes copyable

### ISSUE THREAD

**Issue by a-kenji** [zellij-org/zellij#721]
Make it possible to copy from plugin panes.

**Comment by imsnif** [comment]
As in, copy/paste? What about plugins that use the mouse to do their own thing?

**Comment by a-kenji** [comment]
Yes, as in copy/paste. Maybe have it optional from the plugin side?
I don't know how involved that would be, but for example I can see it being
nice from exploring with strider to be able to copy names of folders and files.
But I also don't see that as a priority.
